src/Bindings/ManualBindings.cpp

```diff
@@ -3954,6 +3954,137 @@ static int tolua_cCompositeChat_UnderlineUrls(lua_State * tolua_S)
 
 
 
+static int tolua_cCuboid_Assign(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	if (!L.CheckParamSelf("cCuboid"))
+	{
+		return 0;
+	}
+
+	cCuboid * self = nullptr;
+	L.GetStackValue(1, self);
+
+	// Check the old coord-based signature:
+	int x1, y1, z1, x2, y2, z2;
+	if (L.GetStackValues(2, x1, y1, z1, x2, y2, z2))
+	{
+		LOGWARNING("cCuboid:Assign(x1, y1, z1, x2, y2, z2) is deprecated, use cCuboid:Assign(Vector3i, Vector3i) instead.");
+		L.LogStackTrace();
+		self->Assign({x1, y1, z1}, {x2, y2, z2});
+		return 0;
+	}
+
+	// Try the (cCuboid) param version:
+	cCuboid * other = nullptr;
+	if (L.GetStackValue(2, other) && (other != nullptr))
+	{
+		self->Assign(*other);
+		return 0;
+	}
+
+	// Try the (Vector3i, Vector3i) param version:
+	Vector3i * pt1 = nullptr;
+	Vector3i * pt2 = nullptr;
+	if (L.GetStackValues(2, pt1, pt2) && (pt1 != nullptr) && (pt2 != nullptr))
+	{
+		self->Assign(*pt1, *pt2);
+		return 0;
+	}
+	tolua_error(tolua_S, "invalid parameter to 'cCuboid:Assign', expected a (cCuboid) or a (Vector3i, Vector3i).", nullptr);
+	return 0;
+}
+
+
+
+
+
+static int tolua_cCuboid_IsInside(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	if (!L.CheckParamSelf("cCuboid"))
+	{
+		return 0;
+	}
+
+	cCuboid * self = nullptr;
+	L.GetStackValue(1, self);
+
+	// Check the old coord-based signature:
+	int x, y, z;
+	if (L.GetStackValues(2, x, y, z))
+	{
+		LOGWARNING("cCuboid:IsInside(x, y, z) is deprecated, use cCuboid:IsInside(Vector3d) instead.");
+		L.LogStackTrace();
+		self->Move({x, y, z});
+		return 0;
+	}
+
+	// Try the (Vector3i) param version:
+	{
+		Vector3i * pt = nullptr;
+		if (L.GetStackValue(2, pt) && (pt != nullptr))
+		{
+			L.Push(self->IsInside(*pt));
+			return 1;
+		}
+	}
+
+	// Try the (Vector3d) param version:
+	{
+		Vector3d * pt = nullptr;
+		if (L.GetStackValue(2, pt) && (pt != nullptr))
+		{
+			L.Push(self->IsInside(*pt));
+			return 1;
+		}
+	}
+	tolua_error(tolua_S, "invalid parameter to 'cCuboid:IsInside', expected a (Vector3i) or (Vector3d).", nullptr);
+	return 0;
+}
+
+
+
+
+
+static int tolua_cCuboid_Move(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	if (!L.CheckParamSelf("cCuboid"))
+	{
+		return 0;
+	}
+
+	cCuboid * self = nullptr;
+	L.GetStackValue(1, self);
+
+	// Check the old coord-based signature:
+	int x, y, z;
+	if (L.GetStackValues(2, x, y, z))
+	{
+		LOGWARNING("cCuboid:Move(x, y, z) is deprecated, use cCuboid:Move(Vector3i) instead.");
+		L.LogStackTrace();
+		self->Move({x, y, z});
+		return 0;
+	}
+
+	Vector3i * offset = nullptr;
+	if (!L.GetStackValue(2, offset) || (offset == nullptr))
+	{
+		tolua_error(tolua_S, "invalid parameter to 'cCuboid:Move', expected a Vector3i.", nullptr);
```

Comment:
```suggestion
		return L.ApiParamError("invalid parameter, expected a Vector3i.");
```
Same for the other functions.